src/otel.rs

```diff
@@ -70,15 +74,17 @@ pub(crate) fn set_otel_parent(req: &ServiceRequest, span: &tracing::Span) {
         feature = "opentelemetry_0_17",
         feature = "opentelemetry_0_18",
         feature = "opentelemetry_0_19",
-        feature = "opentelemetry_0_20"
+        feature = "opentelemetry_0_20",
+        feature = "opentelemetry_0_21"
     )))]
     let trace_id = span.context().span().span_context().trace_id().to_hex();
 
     #[cfg(any(
         feature = "opentelemetry_0_17",
         feature = "opentelemetry_0_18",
         feature = "opentelemetry_0_19",
-        feature = "opentelemetry_0_20"
+        feature = "opentelemetry_0_20",
+        feature = "opentelemetry_0_21"
```

Comment:
I think we can have the trailing comma here (and in every other `cfg(any(...))`) :)
One less change line for future :D

Reply:
Indeed, done. :)